src/_common/sticker/collect/collect.vue

```diff
@@ -3,38 +3,70 @@
 		<div class="-reveal">
 			<div v-if="!canReveal">
 				<app-sticker-card-hidden
-					v-app-tooltip="$gettext(`Not enough progress to unlock sticker`)"
+					v-app-tooltip.touchable="$gettext(`Not enough progress to unlock stickers`)"
 				/>
 			</div>
 			<div v-else-if="isRevealing" class="-card-revealing-outer">
 				<app-sticker-card-hidden class="-card-revealing" />
 			</div>
-			<template v-else-if="isRevealed && purchasedSticker">
+			<template v-else-if="isRevealed && !!purchasedStickers.length">
+				<div
+					v-if="showCollectControls && !!canBuyStickerAmount"
+					class="-revealed-controls-above anim-fade-in"
+				>
+					<app-button primary @click="onClickRepeat(canBuyMultipleAmount)">
+						<translate :translate-params="{ count: canBuyMultipleAmount }">
+							Unlock %{count} More
+						</translate>
+					</app-button>
+				</div>
+				<div v-else class="-revealed-controls-placeholder" />
+
 				<div class="-card-revealed-container">
-					<app-sticker-card
-						class="-card-revealed"
-						:sticker="purchasedSticker"
-						label="+1"
-					/>
-					<div
-						v-if="purchasedSticker.rarity > 0"
-						class="-card-revealed-effect"
-						:class="{
-							'-card-revealed-effect-uncommon': purchasedSticker.rarity === 1,
-							'-card-revealed-effect-rare': purchasedSticker.rarity === 2,
-							'-card-revealed-effect-epic': purchasedSticker.rarity === 3,
-						}"
-					/>
+					<div v-for="{ sticker, sticker_id, count } of limitedStickerDisplay" :key="sticker_id">
+						<app-sticker-card class="-card-revealed" :sticker="sticker" :label="`+${count}`">
+							<div
+								v-if="sticker.rarity > 0"
+								class="-card-revealed-effect"
+								:class="{
+									'-card-revealed-effect-uncommon': sticker.rarity === 1,
+									'-card-revealed-effect-rare': sticker.rarity === 2,
+									'-card-revealed-effect-epic': sticker.rarity === 3,
+								}"
+							/>
+						</app-sticker-card>
+					</div>
 				</div>
+
+				<div
+					v-if="limitedStickerDisplay.length < purchasedStickers.length"
+					class="-revealed-controls-load-more page-cut"
+				>
+					<app-button trans @click="shownRows++">
+						<translate>Load More</translate>
```

Comment:
I think this should say "Show More". I also think it'd be best to just show all when you click this button once, instead of steadily increasing row count.

Reply:
In that case, we could also display "Show All".

Reply:
I changed it to 'Show All" as requested, but I still have it showing one row every purchase - so it will need to be clicked every time if you want to see them all.